perses/rjmc/geometry.py

```diff
@@ -648,15 +649,15 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
 
         state = final_context.getState(getEnergy=True)
         final_context_reduced_potential = beta*state.getPotentialEnergy()
-        final_context_components = [(force, energy*beta) for force, energy in compute_potential_components(final_context, platform=openmm.Platform.getPlatformByName("Reference"))]
-        atoms_with_positions_reduced_potential_components = [(force, energy*beta) for force, energy in compute_potential_components(atoms_with_positions_context, platform=openmm.Platform.getPlatformByName("Reference"))]
```

Comment:
the  default platform for `compute_energy_components` is `utils.get_fastest_platform()` : https://github.com/choderalab/perses/blob/6e5c1932e37e875080528776e554b4fb42efcc85/perses/tests/utils.py#L41
which can return a 'CPU`

Reply:
Should we modify `get_fastest_platform` to specify any requirements (e.g. like double precision is required) so we can return the fastest platform that satisfies those requirements?

Reply:
^ that makes sense